WordPress/Docs/CodeAnalysis/AssignmentInTernaryConditionStandard.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<documentation xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="https://phpcsstandards.github.io/PHPCSDevTools/phpcsdocs.xsd" 
+    title="Assignment In Ternary Condition"
+    >
+    <standard>
+    <![CDATA[
+    Variables should not be assigned in conditional statement of ternary. 
+    Condition must be in parentheses to be checked; ternaries lacking parentheses around condition are skipped. 
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Conditional statement in parentheses and does not include variable assignment.">
+        <![CDATA[
+        $mode = ( $a <em>==</em> 'a' ) ? 'b' : 'c';
```

Comment:
```suggestion
        $mode = ( $a <em>===</em> 'a' ) ? 'b' : 'c';
```

This is another nitpick, but I suggest using a strict comparison instead of a loose comparison in the valid example so that it follows coding best practices.